app/partials/project-card.cjsx

```diff
@@ -12,6 +12,9 @@ counterpart.registerTranslations 'en',
 module.exports = React.createClass
   displayName: 'ProjectCard'
 
+  projectOwner: ->
+    apiClient.type(@props.project.links.owner.type).get(@props.project.links.owner.id)
```

Comment:
When I did project.get 'owner' after doing the include it was still making an http request. I can try fixing this in the json-api-client library since I'll need to do that anyway for the avatar.